examples/beginner/plot_examples.py

```diff
@@ -1,18 +1,15 @@
 #! /usr/bin/env python
 # Check the plot docstring
 
-from sympy import Symbol, exp, sin, cos
+from sympy import Symbols, exp, sin, cos
 from sympy.plotting import (plot, plot_parametric,
                             plot3d_parametric_surface, plot3d_parametric_line,
                             plot3d)
 
 lx = range(5)
 ly = [i**2 for i in lx]
 
-x = Symbol('x')
-y = Symbol('y')
-u = Symbol('u')
-v = Symbol('v')
+x, y, u, v = Symbols('x,y,u,v')
```

Comment:
This is not the right way to use symbols 
Should be like this 
`x, y, u, v = symbols('x y u v')`

Reply:
thank you for your reviews
I changed it and committed